src/pkg/cli/tail.go

```diff
@@ -324,7 +324,7 @@ func tail(ctx context.Context, provider client.Provider, projectName string, opt
 			etag := valueOrDefault(e.Etag, msg.Etag)
 
 			// HACK: skip noisy CI/CD logs (except errors)
-			isInternal := service == "cd" || service == "ci" || service == "kaniko" || service == "fabric" || host == "kaniko" || host == "fabric"
+			isInternal := service == "cd" || service == "ci" || service == "kaniko" || service == "fabric" || host == "kaniko" || host == "fabric" || host == "ecs"
```

Comment:
The reason it's here is because when testing this PR I kept seeing msg from "fabric" and I was confused because there is no Fabric in BYOC. Then I noticed that it's the ECS stream that sets it to Fabric, and the reason for that was for that `--verbose` logic. 

We should revisit all of those, not just `host == "fabric"`. Ideally we remove all the `service == …` ones, since those conflict with user named services.